config/webpack/webpack.config.js

```diff
@@ -9,8 +9,11 @@ const TerserPlugin = require('terser-webpack-plugin');
 const merge = require('webpack-merge');
 const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack-plugin');
 const aggregateTranslations = require('terra-aggregate-translations');
+const webpack = require('webpack');
 const ThemeAggregator = require('../../scripts/aggregate-themes/theme-aggregator');
 
+const base = process.env.BASEPATH;
```

Comment:
Does this default to undefined?

Reply:
If so, lets conditionally add the plugin based on the presence of this variable.